CRM/Member/BAO/Membership.php

```diff
@@ -2398,6 +2398,8 @@ public static function recordMembershipContribution(&$params, $ids = array()) {
       'contribution_recur_id',
       'pan_truncation',
       'card_type_id',
+      'partial_payment_total',
+      'partial_amount_pay',
```

Comment:
Actually, those two are reserved name, already used in Contribution BAO in various places. Should I change there too to ```partial_amount_to_pay```?